### PR TITLE
Bring contents sidebar to the left to make room for wider content 

### PR DESCRIPTION
**Pull request by ascampos**

### CODE REVIEW

File: static/css/core-override.css [hunk 1]
```diff
@@ -90,8 +95,19 @@ section p {
     flex: none;
   }
   header nav.shortcuts {
-    flex: none
+    flex: none;
+    align-items: baseline;
```
Comment (Krief) [inline static/css/core-override.css]: I'm not a fan of what this did. Why are we cramming everything to the left?

File: static/css/core-override.css [hunk 2]
```diff
@@ -90,8 +95,19 @@ section p {
...
+  }
+
+  article > aside .menu ul {
+    border-left: 1px dashed #d4d8de;
+    border-bottom: 1px dashed #d4d8de;
+    padding-left: 15px;
+    color: #6e7477;
```
Comment (Krief) [inline static/css/core-override.css]: I see what this was trying to do, I think. But I don't believe it is succeeding?  The rule 'article > aside .menu .dd-item' seems to take care of it?  

Or maybe 'm not actually clear on what this is going for.

**Comment by Krief** [comment]
The changes needed are very small. After feedback changes are in this should be ready to go!

File: static/css/core-override.css (hunk 2 above)
Comment (ascampos) [reply static/css/core-override.css]: im trying to get the dashed border to appear when you click the dropdown on the inactive items, currently it only shows up after you click on the link

**Pull request merged**